checker/tests/nullness/Issue880.java

```diff
@@ -0,0 +1,22 @@
+// Test case for Issue 880:
```

Comment:
Issue #989 is caused by an annotation in an extends/implements clause, so the test from #880 seems like a good additional test to have.
I agree that we should add an issue that tries getting #876 fixed-up correctly.